solutions/swb-reference/integration-tests/tests/multiStep/awsAccount.test.ts

```diff
@@ -27,61 +25,52 @@ describe('multiStep awsAccount integration test', () => {
   });
 
   test('it works', async () => {
-    const dynamoDbService = new DynamoDBService({
-      region: settings.get('awsRegion'),
-      table: setup.getStackName()
-    });
-    const accountService = new AccountService(dynamoDbService);
+    if (settings.optional('awsAccountId', undefined) !== undefined) {
+      const stackName = setup.getStackName();
 
-    const randomTextGenerator = new RandomTextGenerator(setup.getSettings().get('runId'));
+      const randomTextGenerator = new RandomTextGenerator(setup.getSettings().get('runId'));
 
-    const createAccountParams: CreateAccountRequest = {
-      hostingAccountHandlerRoleArn: settings.get('hostingAccountHandlerRoleArn'),
-      awsAccountId: settings.get('hostAwsAccountId'),
-      envMgmtRoleArn: settings.get('envMgmtRoleArn'),
-      name: randomTextGenerator.getFakeText('fakeName'),
-      externalId: 'workbench'
-    };
+      const awsAccountIdToUse = settings.get('awsAccountId');
 
-    const hostingAwsAccountId = `${resourceTypeToKey.awsAccount}#${createAccountParams.awsAccountId}`;
-    const query = { key: { name: 'pk', value: hostingAwsAccountId } };
-    const ddbEntries = await dynamoDbService.getPaginatedItems(query);
-    if (ddbEntries.data.length > 0) {
-      const accountId = ddbEntries.data[0].accountId.toString();
-      const awsAccountItemKey = {
-        pk: hostingAwsAccountId,
-        sk: `${resourceTypeToKey.account}#${accountId}`
+      const createAccountParams: CreateAccountRequest = {
+        hostingAccountHandlerRoleArn: `arn:aws:iam::${awsAccountIdToUse}:role/${stackName}-hosting-account-role`,
+        awsAccountId: awsAccountIdToUse,
+        envMgmtRoleArn: `arn:aws:iam::${awsAccountIdToUse}:role/${stackName}-env-mgmt`,
+        name: randomTextGenerator.getFakeText('fakeName'),
+        externalId: 'workbench'
```

Comment:
I don't like leaving this in here, we should soon move this to be passed in through integ config settings as well. Maybe add a TODO here